docs/eigenda/operator-guides/run-a-node/registration.md

```diff
@@ -0,0 +1,76 @@
+---
+sidebar_position: 2
+---
+
+# Register Your Operator
+
+Your operator will not begin receiving traffic from the EigenDA disperser until it has registered for one or more quorums with EigenDA. Note, as discussed in [delegation requirements](../requirements/delegation-requirements/), that registration with an EigenDA quorum requires that an operator already be [registered as an operator with EigenLayer](https://docs.eigenlayer.xyz/operator-guides/operator-installation) and to have a minimum amount of stake delegated within each quorum to be registered. 
+
+## Opt-in to an EigenDA Quorum
+
```

Comment:
```suggestion
In order to opt-in into EigenDA as an Operator, you must meet the following delegated TVL requirements:

- Have a minimum of 96 ETH delegated.
- Have more than 1.1x current lowest-stake Operator in the active Operator set. Please see [EigenDA Churn Approver](https://docs.eigenlayer.xyz/operator-guides/avs-installation-and-registration/eigenda-operator-guide#eigenda-churn-approver) for more detail.
- The operator to churn out has less than 10.01% of the total stake

```

Reply:
I've added these requirements to the `delegation requirements` section in order to avoid duplication. That section is linked from here.

Reply:
Nice